### ISSUE TITLE
User Review for cloud-hosted version

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-graphql-client#20]
User Review:

Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough by a non-SME or a new user with a critical and fresh pair of eyes:

- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Report anything that does not work
- [x] Report anything usability issues (confusing or hard to follow)
- [x] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by ReeceNana** [comment]
Reviewed via SN Workbench (Cloud-Hosted)

- Time taken: ~20minutes 
- Each step successful and easy to understand 
- No known logic errors
- Everything works accordingly 
- Guide Achieves basic understanding/working knowledge of the topic from what was understood

- 1 Usability issue:
     - Under 'testing the application', sub-heading 'Running the tests': 

`You can run the Maven verify goal, which compiles the java files, starts the containers, runs the tests, and then          stops the containers.` 

A more detailed, obvious command should be indicated to identify what to enter in order to observe the tests, in line with previous guides/commands given throughout this guide. E.g.:

`mvn verify`

**Comment by shin19991207** [comment]
Fixed issue in https://github.com/OpenLiberty/draft-guide-graphql-client/pull/21
